src/terminal/message_box.rs

```diff
@@ -20,7 +20,7 @@ pub fn dev_alpha_warning() {
             print_centered(width, FEEDBACK_MSG);
             print_centered(width, "");
             print_bottom(width);
-            println!("");
+            println!();
```

Comment:
what's that supposed to do?

Reply:
this code was introduced in #1049 and this diff is just from clippy telling me not to send it an empty &str